pyproject.toml

```
@@ -32,17 +32,18 @@ classifiers = [
   "Topic :: Software Development :: Testing",
 ]
 
+requires-python = ">=3.11,<4.0"
```

```suggestion
requires-python = ">=3.11"
```
Let's keep soft requirements until we met any problems